integrations/tensorflow/bindings/python/pyiree/tf/support/module_utils.py

```diff
@@ -154,22 +110,22 @@ def _incrementally_compile_tf_module(
     A compiled IREE module blob and the path to the compiled VM FlatBuffer if
     artifacts_dir is provided.
   """
-
-  def _compile_module(module, backend_info, exported_names, artifacts_dir):
-    compiler_module = compiler.tf_module_to_compiler_module(module,
-                                                            exported_names,
-                                                            pass_pipeline=())
-    return _incrementally_lower_compiler_module(compiler_module, backend_info,
-                                                artifacts_dir)
-
-  # Avoid the crash reproducer under tests or if the flag is false.
-  # Developers can run tests outside of the test runner (e.g. `bazel run`) to
-  # use the crash reproducer.
-  if (FLAGS.capture_crash_reproducer and not _running_bazel_test()):
-    _compile_module = _setup_mlir_crash_reproducer(_compile_module,
-                                                   artifacts_dir,
-                                                   backend_info.backend_id)
-  return _compile_module(module, backend_info, exported_names, artifacts_dir)
+  output_kwargs = (_get_tf_import_output_kwargs(
+      artifacts_dir,
+      backend_info.backend_id,
+      needs_temp_saved_model_dir=True,
+  ) if artifacts_dir else {})
```

Comment:
```suggestion
  if artifacts_dir:
    output_kwargs = _get_tf_import_output_kwargs(
        artifacts_dir, backend_info.backend_id, needs_temp_saved_model_dir=True)
  else:
    output_kwargs = {}
```
Don't know if it matters, but this follows https://google.github.io/styleguide/pyguide.html#2114-decision with single-assignment.